README.md

```diff
@@ -272,9 +272,9 @@ For information on the classes and types, see the [REFERENCE.md](https://github.
 
 ## Limitations
 
-SQL 2017 and 2019 detection support has been added. This support is limited to functionality already present for other versions. No new SQL 2017 or above specific functionality has been added in this release.
+SQL 2017, 2019 and 2022 detection support has been added. This support is limited to functionality already present for other versions. No new SQL 2017 or above specific functionality has been added in this release.
```

Comment:
`No new SQL 2017 or above specific functionality has been added in this release.`

Do we need this?

Reply:
Updated